### DISCUSSION TITLE
INSS obrigatório para PF

### DISCUSSION THREAD

**Discussion by pauloromeira** [DyegoCosta/trabalhando-remoto#190] [General]
Olá pessoal,

Atualmente trabalho remoto para uma empresa no exterior e recebo (em dólares) como Pessoa Física. Estou na dúvida se devo pagar o INSS além do Carnê Leão.

Em outras issues vi pessoas falando que é facultativo, mas depois de ler este artigo fiquei na dúvida:
https://www.dgabc.com.br/Noticia/2859412/autonomo-que-nao-paga-inss-e-notificado

Já este outro diz que INSS é um tipo de **tributo** e qualquer um que trabalhe remuneradamente, de qualquer forma que seja, deve pagar e que o não pagamento é **crime**:
https://alestrazzi.jusbrasil.com.br/artigos/456628736/nao-quero-pagar-inss-por-causa-da-reforma-posso-quero-meu-dinheiro-de-volta

Eu paguei esse mês o mínimo (11% do salário mínimo = R$ 104,00), mas gostaria de saber se é realmente necessário.

Alguém pode esclarecer?

Abraços.

**Comment by samiraguiar** [comment]
É obrigatório sim, pra todas as pessoas que exercem atividade remunerada no país. É facultativo pra quem não tem renda própria, está desempregado, etc. Dá uma olhada [aqui](https://www.inss.gov.br/orientacoes/tipos-de-filiacao/), você se encaixa em Contribuinte individual.

A legislação disso é o art. 12 da [lei 8.212/91](http://www.planalto.gov.br/ccivil_03/LEIS/L8212compilado.htm), item V, que foi redigido na [lei 9.876/99](http://webcache.googleusercontent.com/search?q=cache:64PBfqFsAooJ:www.planalto.gov.br/ccivil_03/leis/L9876.htm+&cd=1&hl=pt-BR&ct=clnk&gl=br). Diz que o contribuinte individual é segurado obrigatório, sendo contribuinte individual:
> g) quem presta serviço de natureza urbana ou rural, em caráter eventual, a uma ou mais empresas, sem relação de emprego;
> 
> h) a pessoa física que exerce, por conta própria, atividade econômica de natureza urbana, com fins lucrativos ou não;"

Teve até [operação da PF](https://extra.globo.com/noticias/economia/a-partir-de-abril-receita-federal-intensifica-cerco-autonomos-que-devem-ao-inss-22443981.html) pra pegar autônomos que não pagam o imposto.

**Comment by gabrielponto** [comment]
Pessoal, eu acabei deixando isso de lado. Mas realmente é obrigatório. Vão ser 17 meses pagando Carnê Leão mas não paguei INSS. Devo ser cobrado por isso tudo? E ainda, essa cobrança vai vir na forma de 20%?

Eu posso começar a pagar agora e pagar a alíquota de 11% do salário mínimo? Meus rendimentos são providos do exterior.

**Comment by fabioluiz1** [comment]
Vc pode gerar a GPS aqui: http://sal.receita.fazenda.gov.br/PortalSalInternet/faces/pages/calcContribuicoesCI/filiadosApos/selecionarOpcoesCalculoApos.xhtml

Basta vc listar as competências (por exemplo, a competência Nov/18 vence em 15/Dez/18), e os salários de contribuição:

<img width="1207" alt="screen shot 2018-11-29 at 20 38 28" src="https://user-images.githubusercontent.com/508671/49259157-f879f480-f416-11e8-8f24-f0234db4923a.png">

Para entender do que se trata o código do pagamento modalidade 11%, leia o Art 80 da Lei Complementar 123/2006. Basicamente, vc paga menos INSS abrindo mão da opção de aposentadoria por tempo de serviço (não deixe de ler a lei, para embasar corretamente a decisão sobre o seu plano de previdência federal).

O próprio gerador já calcula o valor com multa e juros:

<img width="1214" alt="screen shot 2018-11-29 at 20 38 43" src="https://user-images.githubusercontent.com/508671/49259269-7b02b400-f417-11e8-89bf-cc60e3623efa.png">

Caso queira contribuir normalmente, deve pagar 20% dos ganhos, limitado ao teto de R$ 1.129,16 (quando os rendimentos são iguais ou superiores a R$ 5.645,80).

Lembrando que o INSS também entra como dedução da base de cálculo do Carnê Leão. Ou seja, se vc recebeu 10 mil do exterior e decidir pagar o teto do INSS, o carnê-leão é calculado sobre R$ 8.871,84, ao invés de 10 mil.

**Comment by fabioluiz1** [comment]
Por fim, vc pode também obter mais informações previdenciárias usando o sistema https://meu.inss.gov.br/, por exemplo, vc tem acesso a todo o histórico de contribuições previdenciárias das empresas pelas quais trabalhou.

**Comment by gabrielponto** [comment]
Grande @fabiolnm! Sempre ajudando aqui. Então, como eu tenho um período muito grande a pagar, não gostaria de fazê-lo e sim, ir daqui pra frente.

Mas realmente estou considerando o risco de receber alguma carta com a cobrança. Aí minha dúvida é, se eles realizarem essa cobrança, eu vou poder pagar gerando a GPS ou outra forma diferente? E aí não vou poder escolher a opção de 11%.

**Comment by fabioluiz1** [comment]
Eu pessoalmente recomendo recolher, pois o pagamento de carnê-leão sem pagamento de INSS parece uma maneira muito fácil de ser detectada / cair na malha fina do IRPF.

Além disso, caso o valor que vc faturou não foi tão alto, é capaz de vc receber boa parte de volta, na forma de restituição do imposto de renda.

**Comment by pauloromeira** [comment]
Nossa, muito obrigado pelas informações! @fabiolnm, não sabia que dava pra deduzir do carnê leão... acabei pagando sem deduções :(.

A questã agora é, o que vale mais a pena: pagar R$ 1.129,16 (20% do teto) e ter esse valor deduzido da base de cálculo no carnê leão **ou** R$ 104,94 (11% do mínimo), pagando mais imposto de renda, menos INSS, e aplicar a diferença numa previdência privada? Hahaha...

Obs: engraçado é que no simulador da receita aparece _previdência privada_ em "outras deduções". Mas acredito que esteja errado:
http://www.receita.fazenda.gov.br/aplicacoes/atrjo/simulador/simulador.asp



Abs!

**Comment by fabioluiz1** [comment]
Lance as contribuições no carnê-leão mesmo assim.
Tenho quase certeza que o programa vai abater na sua próxima DARF.
Além disso, quando vc importar os dados do carnê no programa do IRPF, essas contribuições serão usadas para calcular as deduções legais e a restituição do imposto.

**Comment by rabellogp** [comment]
Galera, sempre que leio sobre essa opção do pagamento 11% do mínimo (R$ 104,94) ela é referenciada como uma "mensalidade". Mas como funciona em caso de trabalhos esporádicos? 

Se eu prestar serviços para o exterior umas 2 ou 3 vezes por ano, eu posso utilizar o código do Contribuinte Individual no Plano Simplificado, pagar os R$104 apenas no mês em que realizei o serviço e paguei o Carnê Leão e já está tudo certo? Ou esses planos da previdência devem ser pagos mensalmente independente de eu ter feito algum serviço no mês ou não?

No caso minha única preocupação é não ter problemas posteriormente com juros e multa... Não tenho o menor interesse em "investir" no INSS.

**Comment by gabrweu** [comment]
> <img alt="screen shot 2018-11-29 at 20 38 28" width="1207" src="https://user-images.githubusercontent.com/508671/49259157-f879f480-f416-11e8-8f24-f0234db4923a.png">

@fabiolnm No caso, se eu escolher pagar os 11%, coloco o valor do salário mínimo atual ali no `Salário de Contribuição`? Ou coloco minha renda mesmo.

**Comment by matheustavares** [comment]
> Galera, sempre que leio sobre essa opção do pagamento 11% do mínimo (R$ 104,94) ela é referenciada como uma "mensalidade". Mas como funciona em caso de trabalhos esporádicos?

@rabellogp descobriu alguma coisa sobre essa questão? Estou com a mesma dúvida; liguei no INSS (135) mas o atendente disse que não tinha essa informação e me sugeriu entrar em contato com a receita federal.

**Comment by elMor3no** [comment]
Alguma 

> Galera, sempre que leio sobre essa opção do pagamento 11% do mínimo (R$ 104,94) ela é referenciada como uma "mensalidade". Mas como funciona em caso de trabalhos esporádicos?

Alguém tem alguma informação. To na mesma duvida, pois meus recebimentos são muito esporádicos
